data-prepper-core/src/integrationTest/java/com/amazon/dataprepper/integration/EndToEndRawSpanTest.java

```diff
@@ -107,7 +107,7 @@ public void testPipelineEndToEnd() throws InterruptedException {
         sendExportTraceServiceRequestToSource(DATA_PREPPER_PORT_2, exportTraceServiceRequestTrace2BatchWithRoot);
         sendExportTraceServiceRequestToSource(DATA_PREPPER_PORT_2, exportTraceServiceRequestTrace1BatchNoRoot);
 
-        //Verify data in elasticsearch sink
+        //Verify data in elasticsearch backend
```

Comment:
Replace elasticsearch w/ opensearch here too?

Reply:
It is still using ODFE docker backend so I did not rename it yet